examples/jsm/loaders/LDrawLoader.js

```diff
@@ -2118,8 +2149,7 @@ class LDrawLoader extends Loader {
 
 	getMainEdgeMaterial() {
 
-		const mainMat = this.getMainMaterial();
-		return mainMat && mainMat.userData ? mainMat.userData.edgeMaterial : null;
+		return this.getMaterial( MAIN_EDGE_COLOUR_CODE );
```

Comment:
This previously used to return an "LineBasicMaterial" and now it returns a Mesh material - is this an intended change?